### DISCUSSION TITLE
Biomodel DIY link needs an update

### DISCUSSION THREAD

**Discussion by angel-herraez** [jsme-editor/jsme-editor.github.io#39] [Show and tell]
Hi. Once more, a big thank you for making and keeping JSME in shape.
I notice the page  https://jsme-editor.github.io/jsme-hall-of-fame.html
where the link to my "Do-it-yourself" tool is no longer valid (the redirection to the new server has stopped working)
You may kindly ammend the "Biomodel" link to
Biomodel - Do-It-Yourself Molecules: From 2D to 3D
https://biomodel.uah.es/en/DIY/JSME/draw.en.htm

Thanks

**Comment by jsme-editor** [comment]
Thank you for pointing t his out. Done.
